orm/entities/SubmissionVersion/TallySheetVersion/TallySheetVersionPRE21.py

```diff
@@ -86,9 +89,16 @@ def html(self):
             if row.count is not None:
                 content["totalRejected"] = content["totalRejected"] + row.count
 
+        # convert image to data-uri
+        data_image = ''
```

Comment:
Change this to "logo" and append this as a property to content

Reply:
updated in https://github.com/ECLK/results-tabulation-api/pull/348